outputs.tf

```
@@ -1,5 +1,5 @@
 output "arn" {
-  value       = module.workspace_account.arn
+  value       = module.workspace_account.*.arn
   description = "The workspace user ARN"
```

This may be an empty list which Terraform will complain about, so we can test if it exists and return an empty string instead:

```suggestion
  value       = try(module.workspace_iam_user.0.arn, "")
  description = "The workspace IAM user ARN"
```